### PR TITLE
Fix for --destination option if only a filename is specified

### PR DESCRIPTION
**Pull request by ujjwalwahi**

### CODE REVIEW

**Comment by ujjwalwahi** [comment]
@whimboo 
Submitted PR for --destination issue

**Comment by whimboo** [comment]
Sorry for the late reply but I was out for a month. I will have a look at your PR now.

File: tests/base_scraper/test_base_scraper.py [hunk 1]
```diff
@@ -134,6 +134,13 @@ def test_destination(self):
                                             log_level='ERROR')
         self.assertEqual(scraper.target, destination)
 
+        # destination only has filename
+        scraper = mozdownload.DirectScraper(url=test_url,
+                                      destination=filename,
+                                      version=None,
+                                      log_level='ERROR')
```
Comment (whimboo) [inline tests/base_scraper/test_base_scraper.py]: Just a nit here... Please fix the indentation of the lines 139-141, so they are aligned with `url`. Once this is fixed we can get the PR merged. Thanks a lot!

**Comment by ujjwalwahi** [comment]
@whimboo 
Fixed the indentation

**Comment by whimboo** [comment]
This PR actually fixes issue #267.

**Comment by whimboo** [comment]
Merged as https://github.com/mozilla/mozdownload/commit/ccc2977e238bca4d7099bcc611a2fbad6b2599f2. Thanks @ujjwalwahi for the fix!

**Pull request closed without merging**